sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/LauncherService.java

```diff
@@ -199,10 +200,15 @@ public void buildLauncherJob(Launcher launcher, User user) throws IOException, S
 
         jobParameters.put("overrideFields", args);
 
+        String ciRunId = UUID.randomUUID().toString();
```

Comment:
documentation for this line would be nice so it would be clear while this value is random string